### PR TITLE
feat: update region migration docs

### PR DESCRIPTION
**Pull request by WenyXu**
## What's Changed in this PR

Update region migration docs

## Checklist

- [ ] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: docs/v0.7/zh/user-guide/operations/region-migration.md [hunk 1]
```diff
@@ -6,45 +6,53 @@ Region 迁移允许用户在 Datanode 间移动 Region 数据。
...
+       a.greptime_partition_id as region_id
+from information_schema.partitions a left join information_schema.greptime_region_peers b
+on a.greptime_partition_id = b.region_idwhere a.table_name='migration_target' order by datanode_id asc;
 ```
-例如：我们有以下的 Region 分布
 
-| region id     | leader_peer                                                  | leader_peer_id |
-|---------------|--------------------------------------------------------------|----------------|
-| 4668629450752 | greptimedb-datanode-2.greptimedb-datanode.my-greptimedb:4001 | 2              |
+例如：我们有以下的 Region 分布
```
Comment (tisonkun) [inline docs/v0.7/zh/user-guide/operations/region-migration.md:25]: @nicecui in docs sometimes we use "we", sometimes we use "you". Perhaps we need a guide to use consistent tone.

Comment (nicecui) [reply docs/v0.7/zh/user-guide/operations/region-migration.md:25]: Got it, we will consider this in the documentation style guide.

File: docs/v0.7/en/user-guide/operations/region-migration.md [hunk 2]
```diff
@@ -37,15 +39,21 @@ The region migration won't be performed if the `from_peer_id` equals the `to_pee
...
 ```
 
 | Option           | Description                                                    | Required     |   |
 |------------------|----------------------------------------------------------------|--------------|---|
 | `region_id`      | The region id.                                                 | **Required** |   |
 | `from_peer_id`   | The peer id of the migration source(Datanode).                 | **Required** |   |
 | `to_peer_id`     | The peer id of the migration destination(Datanode).            | **Required** |   |
-| `replay_timeout` | The timeout of replay data. e.g., 1h30m10s                     | **Required** |   |
\ No newline at end of file
+| `replay_timeout` | The timeout of replay data. e.g., 1h30m10s                     |   Optional   |   |
```
Comment (nicecui) [inline docs/v0.7/en/user-guide/operations/region-migration.md:59]: The example 
```
select migrate_region(4398046511104, 1, 2, 60);
```
uses `60` as the replay_timeout, what does that mean?

The second question is, what will happen if the replay data timeout? drop all data or discard remaining data?

Comment (WenyXu) [reply docs/v0.7/en/user-guide/operations/region-migration.md:59]: Good catch!

**Review by nicecui — APPROVED** [review]

File: docs/v0.7/en/user-guide/operations/region-migration.md [hunk 3]
```diff
@@ -37,15 +39,24 @@ The region migration won't be performed if the `from_peer_id` equals the `to_pee
 
 Remember, if you deploy the cluster via the GreptimeDB operator, the `peer_id` of Datanode always starts from 0. For example, if you have a 3 Datanode GreptimeDB cluster, the available `peer_id` will be 0,1,2.
 
-Finally, you can do a Region migration request via the following HTTP request:
+Finally, you can do a Region migration request via the following SQL:
 
 ```bash
```
Comment (tisonkun) [inline docs/v0.7/en/user-guide/operations/region-migration.md:44]:
```suggestion
```sql
```

File: docs/v0.7/en/user-guide/operations/region-migration.md [hunk 4]
```diff
@@ -37,15 +39,24 @@ The region migration won't be performed if the `from_peer_id` equals the `to_pee
...
 | Option           | Description                                                    | Required     |   |
 |------------------|----------------------------------------------------------------|--------------|---|
 | `region_id`      | The region id.                                                 | **Required** |   |
 | `from_peer_id`   | The peer id of the migration source(Datanode).                 | **Required** |   |
 | `to_peer_id`     | The peer id of the migration destination(Datanode).            | **Required** |   |
-| `replay_timeout` | The timeout of replay data. e.g., 1h30m10s                     | **Required** |   |
\ No newline at end of file
+| `replay_timeout` | The timeout(secs) of replay data.                              |   Optional   |   |
+
+Note: If the new Region fails to replay the data within the specified timeout,
```
Comment (tisonkun) [inline docs/v0.7/en/user-guide/operations/region-migration.md:61]: Do we have some Note abomination like:

image

cc @alili 

This is not a blocker for this PR, though.

Comment (tisonkun) [reply docs/v0.7/en/user-guide/operations/region-migration.md:61]: Seems:

````
:::note

...

:::
````
?

**Review by tisonkun — APPROVED** [review]

File: docs/v0.7/en/user-guide/operations/region-migration.md [hunk 5]
```diff
@@ -37,15 +39,24 @@ The region migration won't be performed if the `from_peer_id` equals the `to_pee
...
 ```
 
 | Option           | Description                                                    | Required     |   |
 |------------------|----------------------------------------------------------------|--------------|---|
 | `region_id`      | The region id.                                                 | **Required** |   |
 | `from_peer_id`   | The peer id of the migration source(Datanode).                 | **Required** |   |
 | `to_peer_id`     | The peer id of the migration destination(Datanode).            | **Required** |   |
-| `replay_timeout` | The timeout of replay data. e.g., 1h30m10s                     | **Required** |   |
\ No newline at end of file
+| `replay_timeout` | The timeout(secs) of replay data.                              |   Optional   |   |
```
Comment (nicecui) [inline docs/v0.7/en/user-guide/operations/region-migration.md:59]:
```suggestion
| `replay_timeout` | The timeout(secs) of replay data. If the new Region fails to replay the data within the specified timeout, 
the migration will fail, however the data in the old Region will not be lost.                             |   Optional   |   |
```

File: docs/v0.7/en/user-guide/operations/region-migration.md [hunk 6]
```diff
@@ -37,15 +39,24 @@ The region migration won't be performed if the `from_peer_id` equals the `to_pee
...
+
+Note: If the new Region fails to replay the data within the specified timeout,
```
Comment (nicecui) [inline docs/v0.7/en/user-guide/operations/region-migration.md:61]:
```suggestion

```

File: docs/v0.7/en/user-guide/operations/region-migration.md [hunk 7]
```diff
@@ -37,15 +39,24 @@ The region migration won't be performed if the `from_peer_id` equals the `to_pee
...
 |------------------|----------------------------------------------------------------|--------------|---|
 | `region_id`      | The region id.                                                 | **Required** |   |
 | `from_peer_id`   | The peer id of the migration source(Datanode).                 | **Required** |   |
 | `to_peer_id`     | The peer id of the migration destination(Datanode).            | **Required** |   |
-| `replay_timeout` | The timeout of replay data. e.g., 1h30m10s                     | **Required** |   |
\ No newline at end of file
+| `replay_timeout` | The timeout(secs) of replay data.                              |   Optional   |   |
+
+Note: If the new Region fails to replay the data within the specified timeout, 
+the migration will fail; the data in the old Region will not be lost.
```
Comment (nicecui) [inline docs/v0.7/en/user-guide/operations/region-migration.md:62]:
```suggestion

```

File: docs/v0.7/zh/user-guide/operations/region-migration.md [hunk 8]
```diff
@@ -6,45 +6,55 @@ Region 迁移允许用户在 Datanode 间移动 Region 数据。
...
 ```
 
 | Option           | Description                                                    | Required     |   |
 |------------------|----------------------------------------------------------------|--------------|---|
 | `region_id`      | Region Id                                                      | **Required** |   |
 | `from_peer_id`   | 迁移起始节点(Datanode) 的 peer id。                               | **Required** |   |
 | `to_peer_id`     | 迁移目标节点(Datanode) 的 peer id。                               | **Required** |   |
-| `replay_timeout` | 迁移时回放数据的超时时间，例如：1h30m10s                             | **Required** |   |
\ No newline at end of file
+| `replay_timeout` | 迁移时回放数据的超时时间（单位：秒）                                 |   Optional   |   |
```
Comment (nicecui) [inline docs/v0.7/zh/user-guide/operations/region-migration.md:58]:
```suggestion
| `replay_timeout` | 迁移时回放数据的超时时间（单位：秒）。如果新 Region 未能在指定时间内回放数据，迁移将失败，旧 Region 中的数据不会丢失。                               |   Optional   |   |
```

File: docs/v0.7/zh/user-guide/operations/region-migration.md [hunk 9]
```diff
@@ -6,45 +6,55 @@ Region 迁移允许用户在 Datanode 间移动 Region 数据。
...
+
+注意：如果新 Region 未能在指定时间内回放数据，迁移将失败；旧 Region 中的数据不会丢失。
```
Comment (nicecui) [inline docs/v0.7/zh/user-guide/operations/region-migration.md:60]:
```suggestion

```

File: docs/v0.7/en/user-guide/operations/region-migration.md (hunk 4 above)
Comment (WenyXu) [reply docs/v0.7/en/user-guide/operations/region-migration.md:61]: let me fix this

**Pull request merged**